### ISSUE TITLE
effective Higgs couplings for UFO model

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#402]
SARAH generated UFO has loop induced couplings of Higgs. They are read from the `EFFHIGGSCOUPLINGS` block of SLHA output. I'd like to add it to the output unless someone objects.

**Comment by uukhas** [comment]
@wkotlarski, what could be the problem if you add it?

**Comment by wkotlarski** [comment]
Someone might not like it :shrug:

**Comment by uukhas** [comment]
If the output it "orthogonal"  to the existing one, then I am fine with the change.

**Comment by Expander** [comment]
Just to be clear: Is the output that you propose orthogonal to existing output of FS? I'd agree, that if it is something new, I'm not against adding it.

**Comment by wkotlarski** [comment]
Yes. It would also be used only when reading model parameters into UFO model in Madgraph, etc.

**Comment by pathron** [comment]
I doubt I am against but sorry I still don't fully get what the proposal is yet. The point is that the sarah generated UFO files tell madgraph to expect the EFFHIGGSCOUPLINGS block?  So now you need this block to be in the SLHA? 

So then you just add it to the FS SLHA file as a new block?  

But is this block / the definition of the  effective couplings different to the old ones we had prior to FlexibleDecay and does it resolve some of the problems you had with them regarding the definition / meaning.

**Comment by wkotlarski** [comment]
It should be similar to the one we had before but not necessarily identical. The potential difference is that I have to match the convention or SARAH generated UFO. Florian does this
```
WriteString[sphenoInOut,"facPP = Alpha*Sqrt(2._dp*G_F/sqrt(2._dp))/(2._dp*Pi) \n"];
...
WriteString[sphenoInOut,"Write(io_L,121) CurrentPDG3(1), CurrentPDG3(2), CurrentPDG3(3), Abs(CoupHPP"<>addGen<>")*facPP, \" # H-Photon-Photon \" \n"];
...
```
The `Abs` is fishy cause it makes interference contributions wrong so I'll skip it.

**Comment by pathron** [comment]
I would have expected this to match our old conventions since we added those while working with Florian.

What about the problems you had before with the definitions / meaning, they don't apply here?

**Comment by wkotlarski** [comment]
I'm sorry but I don't know which problems you're referring to.

**Comment by wkotlarski** [comment]
This is an approximation which for tree-level monte carlo should be ok-ish enough. This is also what HiggsBounds/HiggsSignals would do if you run them in the effective coupling mode.
